static/app/views/settings/project/appStoreConnectContext.tsx

```diff
@@ -2,11 +2,11 @@ import {createContext, useEffect, useState} from 'react';
 
```

Comment:
can we pls delete this file? looks like it's not being used

Reply:
if this gets removed i think this leaves `static/app/components/projects/appStoreConnectContext/utils.tsx` in a strange place. where do you suppose these files should sit instead? would `static/app/components/modals/debugFileCustomRepository/appStoreConnect/utils.tsx` be a good new home?

Reply:
we have 2 appStoreConnectContext... one is the file `'app/components/projects/appStoreConnectContext'` which we use in many different places and the other one is `static/app/components/projects/appStoreConnectContext` which apparently is not being used... but please do not worry... the used file will be removed in this PR https://github.com/getsentry/sentry/pull/29843/ 😉